engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -292,14 +292,13 @@ public void onPlayerSpawn(OnPlayerSpawnedEvent event, EntityRef character) {
     }
 
     @ReceiveEvent(components = CharacterComponent.class)
-    public void onMouseX(MouseXAxisEvent event, EntityRef entity) {
-        lookYawDelta = event.getValue();
-        event.consume();
-    }
-
-    @ReceiveEvent(components = CharacterComponent.class)
-    public void onMouseY(MouseYAxisEvent event, EntityRef entity) {
-        lookPitchDelta = event.getValue();
+    public void onMouseX(MouseAxisEvent event, EntityRef entity) {
+        MouseAxis axis = event.getMouseAxis();
```

Comment:
I might be missing something, but shouldn't this just be onMouse(), since we're handling both X and Y axis here?